core/src/main/scala/com/criteo/cuttle/App.scala

```diff
@@ -364,6 +364,16 @@ private[cuttle] case class App[S <: Scheduling](project: CuttleProject[S], execu
       IO.pure(Ok)
     }
 
+    case POST at url"/api/executions/forceSuccess?jobs=$jobs" => { implicit user =>
```

Comment:
I do not think this is the endpoint we need. I was thinking more of an endpoint to force success on a single execution. Here you are forcing success on a potentially big number of executions which, I think, should be avoided.

Reply:
I will make the change. That makes sense.